src/WarcraftLegacies.Source/Quests/Skywall/QuestFirelandInvasion.cs

```diff
@@ -66,7 +66,7 @@ protected override void OnComplete(Faction completingFaction)
         SpawnArmies(completingFaction);
 
       ResetVictimControlPointLevel();
-      PresentInvasionDialogs();
+      //PresentInvasionDialogs();
```

Comment:
This is such an easy thing to fix I probably wouldn't even bother commenting it out, but your call.

Reply:
Intention is to be able to release this and now rush or be blocked by Sebas updating the invasion